Resources/Prototypes/Recipes/Reactions/medicine.yml

```diff
@@ -310,3 +310,21 @@
       amount: 1
   products:
     Spaceacillin: 2
+    
+- type: reaction
+  id: Conscizine
+  reactants:
+    Epinephrine:
+      amount: 1
+    Tricordrazine:
+      amount: 1
+    MindbreakerToxin:
+      amount: 1
+    Theobromine:
+      amount: 1
+    Spaceacillin:
+      amount: 1
+    Clonexadone:
+      amount: 1
```

Comment:
this need to be harder to make in the sense that its ingredients are hard to acquire, preferably requiring some chemicals that can only be acquired in salvage (or nocturine?), not just by having a lot of ingredients

Reply:
if you dont want to do that in this PR then imo the recipe should be removed until that can be added